### ISSUE TITLE
Issue regarding resizing image

### ISSUE THREAD

**Issue by arkchkp** [Jojodicus/qr2eascii#3]
Hi team!!

We are testing this repo and we are facing issues. It seems it is related to the resizing.

Once we run:

```
python3 convert.py --invert
```

We get this error:

```
Enter data to encode: https://www.test.com
Traceback (most recent call last):
  File "/home/iberia/qr2eascii/convert.py", line 66, in <module>
    image = image.resize((width // scale, height // scale), Image.Resampling.NEAREST)
  File "/usr/lib/python3/dist-packages/PIL/Image.py", line 65, in __getattr__
    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
AttributeError: module 'PIL.Image' has no attribute 'Resampling'
```

Any idea about what I am doing wrong?

Thanks!

**Comment by Jojodicus** [comment]
hi, I'm guessing you are using an old version of `pillow`. I can reproduce this with 9.0.1 on a Mint 21 ISO.
Back then, the people over at pillow just couldn't decide on how they want to handle their constants and randomly removed some options. Added a small fix for those affected version in a7dfb51, so it should work now regardless of the environment.

**Comment by arkchkp** [comment]
It works! :D 

Thank you very much!
